modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportObjectDefinitionTest.java

```diff
@@ -178,6 +214,19 @@ private byte[] _getContent(String boundary, byte[] bytes, String fileName) {
 		return ArrayUtil.append(start.getBytes(), bytes, end.getBytes());
 	}
 
+	private ObjectDefinition _getObjectDefinitionByName(
+			String name, ObjectDefinitionResource objectDefinitionResource)
+		throws Exception {
+
+		Page<ObjectDefinition> page =
+			objectDefinitionResource.getObjectDefinitionsPage(
+				name, null, null, Pagination.of(1, 1), null);
+
+		List<ObjectDefinition> items = (List<ObjectDefinition>)page.getItems();
+
+		return items.get(0);
+	}
```

Comment:
@pedro-oliveira446 after you join the methods below you can put this snippet of code inside the new method since we will be using in only one place.